genstudio-create-context-addon/package.json

```diff
@@ -4,22 +4,15 @@
   "private": true,
   "dependencies": {
     "@adobe/aem-cf-editor-ui-ext-tpl": "^2.1.0",
-    "@adobe/aio-sdk": "^6.0.0",
-    "@adobe/exc-app": "^0.2.21",
     "@adobe/genstudio-uix-sdk": "^1.6.5",
     "@adobe/react-spectrum": "^3.4.0",
     "@adobe/uix-guest": "^0.8.0",
-    "@react-spectrum/list": "^3.0.0-rc.0",
-    "@spectrum-icons/workflow": "^3.2.0",
     "chalk": "^4",
```

Comment:
interesting that we use chalk... that should just be for logging?

Reply:
oh i think the deploy hooks use it... i think it can be moved to dev dep

Reply:
looks like so. updated